Sources/Purchasing/EntitlementInfo.swift

```diff
@@ -133,6 +133,11 @@ extension PeriodType: DefaultValueProvider {
      */
     @objc public var productIdentifier: String { self.contents.productIdentifier }
 
+    /**
+     The product plan identifier that unlocked this entitlement
```

Comment:
Can you add a note that this is just for Android? In fact I wonder if this even needs to be exposed?

Reply:
Note that if we _do_ need to expose it you need to change this from `fix` to `feat` and add this new method to the API testers.

Reply:
I made it internal! Doesn't need to be public at all yet (unless some users think it should be 🤷‍♂️)